src/NuGet.Clients/NuGet.CommandLine/Common/Console.cs

```diff
@@ -395,7 +395,7 @@ public override void Log(ILogMessage message)
                 }
                 else if (message.Level == LogLevel.Error)
                 {
-                    WriteError(message.FormatWithCode());
+                    WriteError("error " + message.FormatWithCode());
```

Comment:
the error word should  be a string in Resources.resx so it can be localized

Reply:
Fixed